packages/shared/src/components/feeds/FeedSettings/FeedSettingsEditHeader.tsx

```diff
@@ -18,7 +18,7 @@ const SaveButton = ({ activeView }: { activeView: string }): ReactElement => {
     FeedSettingsEditContext,
   );
 
-  if (activeView !== 'General') {
+  if (activeView !== 'General' && activeView !== 'Filters') {
```

Comment:
Do we have `General` and `Filters` as enums somewhere? So we don't get typos if we need more in the future?